nixos/modules/hardware/network/intel-2200bg.nix

```diff
@@ -23,7 +23,7 @@
 
   config = lib.mkIf config.networking.enableIntel2200BGFirmware {
 
-    hardware.enableAllFirmware = true;
+    hardware.firmware = [ pkgs.intel2200BGFirmware ];
```

Comment:
What approach makes more sense in your opinion? I think that adding to `enableAllFirmware` would make more sense.

Generally shouldn't all `enable*Firmware` just append to the list?